app/src/main/java/com/techyourchance/coroutines/common/dependencyinjection/ActivityCompositionRoot.kt

```diff
@@ -84,4 +85,6 @@ class ActivityCompositionRoot(
 
     val loginUseCaseUncaughtException get() = LoginUseCaseUncaughtException(loginEndpointUncaughtException, userStateManager)
 
+    val emulateNetworkCallUseCase get() = EmulateNetworkCallUseCase()
```

Comment:
Out of curiosity, can you explain how your fragment use this ActivityCompositionRoot?